front/public/locales/en/errors.json

```diff
@@ -153,6 +153,12 @@
     "stdcm": {
       "InfraNotFound": "Infra {{infra_id}} does not exist"
     },
+    "stdcm_v2": {
+      "InfraNotFound": "Infra {{infra_id}} does not exist",
```

Comment:
For some english trad, we use `Infrastructure` in full letters and sometimes we don't, we should use only one of these.

Reply:
Il will use `Infrastructure` for this error and I will make a PR to replace `Infra` by `Infrastructure` everywhere else :